src/commands/githubActions/endToEndTest/testCheckers.ts

```diff
@@ -116,10 +116,42 @@ async function ensureNoErrorLogs(
 }
 
 /**
- * Test that the validators are attesting after a peri
+ * Test that the validators are attesting after 16 minutes
+ * 
  */
-async function attestanceProof(network: Network): Promise<void> {
-  if (network === "mainnet") return;
-  // TODO
+export async function attestanceProof(network: Network): Promise<void> {
+  // Check if the network is mainnet or prater
+  if (network === "mainnet") {
+    // TODO
+    return;
+  } else if (network === "prater") {
+    let iterationCount = 1;
+    console.log(chalk.grey(`  - Checking if validator is active`));
+
+    while (iterationCount <= 9) {
+      await new Promise(resolve => setTimeout(resolve, 2 * 60 * 1000)); // Wait for 2 minutes before each iteration
+      const response = await got(`https://prater.beaconcha.in/api/v1/validator/${process.env.VALIDATOR_INDEX}`, {
+      headers: {
+        Accept: 'application/json'
+      },
+      responseType: 'json',
+      throwHttpErrors: false
+      });
+
+      if (response.statusCode !== 200) throw Error(`Error while fetching validator data. Beaconcha.in returned ${response.statusCode}`);
+
+      const data = response.body as ValidatorData
+      if (data.data.status === "active_online") {
+        console.log(chalk.green(`  ✓ Validator is active`));
+        return; // Exit the loop if the validator is active
+      } else {
+        console.log(chalk.yellow(`  - Validator is not active yet. Retrying (minutes passed: ${iterationCount*2})`));
+      }
+      iterationCount++;
```

Comment:
You coud use a `for` loop to avoid this here, but not sure if it is a good practice for this case as you are probably going to break the loop with the `return` before it reaches the end